frontend/src/Components/loginForm.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+
+
+export const LoginForm = () => {
+    const [username, setUsername] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+          axios.post("http://localhost:8000/login/password", {
+            username,
+            password,
+          })
+          .then(response => {
+            console.log(response.data);
+          })
+          .catch(error => {
+            console.error(error);
+          });
```

Comment:
i think this works just fine! but let's take a stab at converting this into async/await style js rather than promises

https://rapidapi.com/guides/axios-async-await

Reply:
Okay fixed! By the reading its to make the code look more clean right?